util/expensivequery/expensivequery.go

```diff
@@ -75,6 +75,12 @@ func (eqh *Handle) Run() {
 				}
 			}
 			threshold = atomic.LoadUint64(&variable.ExpensiveQueryTimeThreshold)
+
+			ratio := variable.MemoryUsageAlarmRatio.Load()
+			if ratio != record.serverMemoryQuotaRatio {
+				record.serverMemoryQuotaRatio = ratio
```

Comment:
`record.serverMemoryQuotaRatio = ratio.variable.MemoryUsageAlarmRatio.Load()` is enough?